Game/MainScene.gd

```diff
@@ -132,18 +138,12 @@ func endCurrentScene():
 	var currentScene = getCurrentScene()
 	if(currentScene != null):
 		currentScene.endScene()
-
-func _on_GameUI_on_option_button(method, args):
-	pickOption(method, args)
 	
 func pickOption(method, args):
 	GM.main.clearMessages()
 	GlobalTooltip.resetTooltips()
 	
-	if(GM.ES.checkButtonInput(method, args)):
-		pass
-		
-	elif(sceneStack.size() > 0):
+	if !(GM.ES.checkButtonInput(method, args) && (sceneStack.size() > 0)):
```

Comment:
Woops...
It's should looks like this: if (!GM.ES.checkButtonInput(method, args) && (sceneStack.size() > 0))